rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/project/ProjectController.java

```diff
@@ -2511,6 +2511,25 @@ public void getUserProjectCount(HttpServletResponse response) throws TException
         }
     }
 
+    @Operation(
+            description = "Get the default license info header text.",
+            tags = {"Projects"}
+    )
+    @RequestMapping(value = PROJECTS_URL + "/licenseInfoHeader", method = RequestMethod.GET)
+    public void getLicenseInfoheaderText(HttpServletResponse response) throws TException {
+        User sw360User = restControllerHelper.getSw360UserFromAuthentication();
+        try {
+            response.setContentType("application/json; charset=UTF-8");
+            response.setCharacterEncoding("UTF-8");
+            String text = projectService.getLicenseInfoHeaderText();
```

Comment:
Please use better name for example.
```suggestion
            String licenseHeaderText = projectService.getLicenseInfoHeaderText();
```